### DISCUSSION TITLE
I forgot email that I used for my org

### DISCUSSION THREAD

**Discussion by vsamofal** [community/community#66413] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

https://github.com/softkit

This one, I believe I've registered it many years ago, but I don't remember on which email. 
Now we do use this one https://github.com/softkitit
but would like to get the original one back if that's possible. 

is it possible to get it somehow?

**Comment by SreejithMVarma** [comment, marked as answer]
### CASE 1:

**If you have access** to the email address associated with the original account,

1. **Password Reset:**
   - Go to the GitHub login page: https://github.com/login
   - Click on the "Forgot password?" or "Trouble logging in?" link.

2. **Provide Username or Email:**
   - Enter the username or the email address you think might be associated with your original account (even if you're not sure). GitHub will then send you an email to that address with instructions on how to reset your password.

3. **Check Email Inbox:**
   - Check the email inbox for the address you provided in step 2. You should receive an email from GitHub with a link to reset your password.

4. **Reset Password:**
   - Click on the link in the email. This will take you to a page where you can create a new password for your original GitHub account.

5. **Access Your Account:**
   - Once you've reset the password, use it to log in to your original GitHub account at https://github.com/softkit.


### CASE 2:


**If you no longer have access** to the email address associated with the original account, 
In this situation, it becomes challenging to recover your GitHub account on your own. To recover your account, you should consider the following options:

1. Email Recovery Options: If you have any old records or documents that might contain the email address you used for GitHub, try to find them. It could be an old registration confirmation email from GitHub or any related documentation.

2. Contact GitHub Support: If you cannot determine the email address or have no way of accessing it, you should contact GitHub Support directly. They may ask you for some identifying information to verify your ownership of the account. Provide them with as much information as possible, such as your username, full name, and any other details you remember about the account. They will assist you in the recovery process.

3. Provide Proof of Ownership: If you have any documents, code, or other content associated with the account that can demonstrate your ownership, be prepared to provide that as evidence to GitHub Support.

Hope it helps! ❤️

**Comment by thanmaisai** [comment]
If you've forgotten the email associated with your GitHub account and are unable to provide sufficient proof of ownership, there are a few more options to explore:

### Check Your Email Accounts:

Go through your email accounts to see if you can find any old emails from GitHub. These emails may contain information about your account or the associated email address.
### Check for Password Managers or Browsers Saved Passwords:

If you've ever saved your GitHub login information in a password manager or in your browser's saved passwords, you may be able to retrieve the email address associated with your account from there.
### Contact GitHub Support with More Information:

If you can provide additional details about your account, such as specific repositories you owned or contributed to, recent activity, or any connections you had on GitHub, include this information in your support request. The more details you can provide, the better your chances of recovering your account.
### Try Different Email Addresses:

If you have multiple email addresses, try reaching out to GitHub support using each of them. One of them might be the correct email associated with your account.
### Reach Out on Social Media:

You can also try reaching out to GitHub support through their social media channels like Twitter. Sometimes, they may be able to provide additional assistance.

Remember that account recovery can be a complex and time-consuming process, and there are no guarantees of success. It's important to be patient and provide as much information as possible to GitHub support to help them verify your ownership of the account. The more evidence you can provide, the better your chances of regaining access to your original account.

**Reply by donakhseputa** [reply]
why github not sent recovery email just by input username and show redacted email, for example my username loremipsum, and github just sent emai to loremipsum123@example.com, but redacted to lor****23@example.com. that might be help many people to find their email.

**Comment by Ranimto** [comment]
I forgot my email that is connected to my other github account but i have the password and the username correct but i can t verify it via email how can i do to get my old account how to contact the support team ?

**Reply by ghost** [reply]
Have same issue. Did you resolve your issue @Ranimto ?

**Reply by Khant-Nyar** [reply]
also me

**Comment by ghost** [comment]
I lost my 2FA recovery code

**Comment by yokeshbaskaran** [comment]
Check ur email. 
Check ur settings. 
Google it for solution or watch youtube for clear solution.

**Comment by Taxtander** [comment]
Check ur email.
Check ur settings.
Google it for solution or watch youtube for clear solution.
